src/runtime/opencl/opencl_module.cc

```diff
@@ -353,9 +357,10 @@ PackedFunc OpenCLModuleNode::GetFunction(const String& name,
 }
 
 Module OpenCLModuleCreate(std::string data, std::string fmt,
-                          std::unordered_map<std::string, FunctionInfo> fmap, std::string source) {
+                          std::unordered_map<std::string, FunctionInfo> fmap, std::string source,
+                          bool dry_run) {
   auto n = make_object<OpenCLModuleNode>(data, fmt, fmap, source);
-  n->Init();
+  n->Init(dry_run);
```

Comment:
`dry_run` name would assume that we need to have such object and call some API, but results of these calls are not important.

On the other hand, I would assume that no API is called and creation and initialization of OpenCL is just a resuilt of bad design. So, we need either not to create instance of this class or it we have several purposes inside this class and it dedicate for both runtime and some service capabilities, we need to rename this parameter

Reply:
Thank you for your comment! Previous implementation was refactored and now we don't need this parameter.